pychunkedgraph/meshing/meshgen.py

```diff
@@ -296,13 +296,15 @@ def calculate_stop_layer(cg, chunk_id):
         for y in range(chunk_coords[1], chunk_coords[1] + 2):
             for z in range(chunk_coords[2], chunk_coords[2] + 2):
 
-                # Chunk id
-                neigh_chunk_id = cg.get_chunk_id(x=x, y=y, z=z, layer=chunk_layer)
-                neigh_chunk_ids.append(neigh_chunk_id)
-
-                # Get parent chunk ids
-                parent_chunk_ids = cg.get_parent_chunk_ids(neigh_chunk_id)
-                neigh_parent_chunk_ids.append(parent_chunk_ids)
+                try:
+                    # Chunk id
+                    neigh_chunk_id = cg.get_chunk_id(x=x, y=y, z=z, layer=chunk_layer)
+                    # Get parent chunk ids
+                    parent_chunk_ids = cg.get_parent_chunk_ids(neigh_chunk_id)
+                    neigh_chunk_ids.append(neigh_chunk_id)
+                    neigh_parent_chunk_ids.append(parent_chunk_ids)
+                except:
+                    pass
```

Comment:
why does it fail here when it fails. Could this be added to the except?

Reply:
it's a rare edge case that never came up before (the neighboring chunk is outside the dataset and the # of chunks in the widest dimension is equal to a power of two). i added a comment to the except.